examples/sysctrl/hello-sysctrl/examples/init_hpc.rs

```diff
@@ -99,22 +101,68 @@ fn main() -> ! {
 
     print_example_name!();
 
+    // Enable SDRAM
+    let ddr_mode = 0b1 << 1;
+    let axi_enable = 0b1 << 2;
```

Comment:
! There's a problem here. `0b1 << 1` is not `1` but `2` and `0b1 << 2` is 4.

Reply:
Oh that's true, you wrote it correctly in the previous error, but I failed to read it correctly. Fixed.